io.github.ferraridamiano.ConverterNOW.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Version=3.1
```

Comment:
This is not what you think it is:
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html

> Version of the Desktop Entry Specification that the desktop entry conforms with. Entries that confirm with this version of the specification should use 1.5. Note that the version field is not required to be present. 

In short `3.1` is likely not correct.